include/NAS2D/Renderer/RendererOpenGL.h

```diff
@@ -11,6 +11,9 @@
 
 #include "Renderer.h"
 
+#include "Point.h"
+#include "Vector.h"
```

Comment:
No problem with this, just wanted to point out these are included by the base class. That is necessarily true, since the base class has method signatures that use them. I'm uncertain if it's more proper to include these or leave them out for derived classes. Might be a point worth discussing.